sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -788,7 +788,7 @@ booleanValue
 
 interval
     : negativeSign=MINUS? INTERVAL (errorCapturingMultiUnitsInterval | errorCapturingUnitToUnitInterval)?
-    | {ansi}? (errorCapturingMultiUnitsInterval | errorCapturingUnitToUnitInterval)
+    | {use_SQL_standard_keywords}? (errorCapturingMultiUnitsInterval | errorCapturingUnitToUnitInterval)
```

Comment:
@maropu maybe we should have a separated config to allow not specifying the leading INTERVAL. pgsql requires the leading INTERVAL
```
cloud0fan=# select interval '1' day;
 interval 
----------
 1 day
(1 row)

cloud0fan=# select '1' day;
ERROR:  syntax error at or near "day"
LINE 1: select '1' day;
```

Reply:
Ur, it looks reasonable to me. I'll file jira for that.

Reply:
Ur, I see. I just followed the behaviour of the standard, hive, and mysql. But, it looks reasonable to me. So, I'll file jira for that.

Reply:
I checked SQL standard:
```
<interval literal> ::=
  INTERVAL [ <sign> ] <interval string> <interval qualifier>

<interval string> ::=
  <quote> <unquoted interval string> <quote>
```

Spark SQL doesn't follow it completely as we allow `interval 1 year 2 days`, but `INTERVAL` should be required under ansi mode.